sources.json

```diff
@@ -242,6 +242,78 @@
     "tags": ["odata", "promise"],
     "license": "mit"
   },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-error-collector",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["error","collector","exception","log","event"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-smart-mockserver",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
```

Comment:
@mauriciolauffer @marianfoo : would be rather a `library`?

Reply:
`library` isn't a valid enum:
https://github.com/ui5-community/bestofui5-data/blob/5860fd33a980bcdc8c23fb3b7bf25d7c36607ebe/src/types.d.ts#L1-L8